nullaway/src/test/java/com/uber/nullaway/NullAwayContractsTests.java

```diff
@@ -68,6 +68,25 @@ public void basicContractAnnotation() {
         .doTest();
   }
 
+  @Test
+  public void contractArgsEmpty() {
```

Comment:
Rather than this test, would it be possible to write a separate test that directly calls into the APIs of `ContractUtils` and asserts that the appropriate string is parsed as intended?

Reply:
Sure thing